Comparison Engine/Comparison Engine/GoogleMap/MapController.cs

```diff
@@ -47,11 +49,51 @@ public void RemoveOverlays(GMapControl map)
             }
         }
 
+        // puts markers on all bars within a wanted radius which have a specific drink and notes it's price in those bars
+        //can't really test, cuz designer is broken on my side, so the string formatting might need changing
+        //might need to adjust map by zooming out in the future
+        public void ShowBarsWithDrink(GMapControl map, Drink specificDrink, double prefDistance, string currentAddress, List<Bar> barList)
+        {
+            var barIDs = specificDrink.drinkLocations.Keys.ToArray();
+
+            foreach (int ID in barIDs)
+            {
+                var bar = barList[ID];
+
+                MarkBarsInRadius(map, prefDistance, currentAddress, bar, $"{bar.barName} \n {bar.location} \n {specificDrink.drinkLocations[ID]}");
+            }
+
+            ShowMapToPoint(map, GetPointFromAddress(currentAddress));
+        }
+
+        //puts markers on all bars within a wanted radius with addresses and names
+        //can't really test, cuz designer is broken on my side, so the string formatting might need changing
+        //might need to adjust map by zooming out in the future
+        public void ShowNearBars(GMapControl map, double prefDistance, string currentAddress, List<Bar> barList)
+        {
+            foreach (Bar bar in barList)
+            {
+                MarkBarsInRadius(map, prefDistance, currentAddress, bar, $"{bar.barName} \n {bar.location}");
+            }
+
+            ShowMapToPoint(map, GetPointFromAddress(currentAddress));
+        }
+
+        private void MarkBarsInRadius(GMapControl map, double prefDistance, string currentAddress, Bar bar, string markerString)
```

Comment:
Technically should probably be named "MarkBarInRadius" since it marks a single bar

Reply:
That's absolutely right, thank you for noticing that.  Probably forgot to adjust names when renaming and restructuring functions